xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/usersandgroups/usersandgroups.js

```diff
@@ -89,6 +99,8 @@ window.MSCheckbox = Class.create({
       delete this.table.fetchedRows[this.idx];
     }
     this.draw(this.state);
+    /*After changing the image through an interaction, put focus back on it.*/
```

Comment:
This is explaining what it does but not why. In comments, it's always better to explain why we do something (more useful when another coder reads it or when you read it back a few years after :)).